client/src/components/Signup/SignupForm.jsx

```diff
@@ -59,7 +59,7 @@ function SignupForm() {
 
   return (
     <div>
-      {signUpError ? <Alert variant="danger">{signUpError}</Alert> : undefined}
+      {signUpError ? <Alert variant="danger">{"Please fill out all of the information"}</Alert> : undefined}
```

Comment:
This isn't necessarily the only error that would pop up. If the email already exists, it should not allow the signup to continue and display an appropriate error message.